tests/containers_tests.py

```diff
@@ -117,10 +118,14 @@ def test_different_barcodes_are_created_when_not_provided(self):
     self.assertNotEqual(bc1, bc2)
 
   def test_cannot_supply_aker_barcode(self):
-    data = valid_container_params({'barcode':'aker-abc'})
-    response, status = self.post('/containers', data=data)
-    self.assertValidationErrorStatus(status)
-    self.assertValidationError(response, { 'barcode': 'AKER barcode not permitted: aker-abc'})
+    data = valid_container_params()
+
+    response, status = self.get(self.domain['containers']['url']+'/schema')
+    if response['barcode']['non_aker_barcode']:
```

Comment:
I don't understand this. Won't this never be true? `non_aker_barcode` is not an attribute of container